packages/shared/src/aws/tile.metadata.imagery.ts

```diff
@@ -33,19 +33,20 @@ export function compareImageSets(a: TileSetRuleImagery, b: TileSetRuleImagery):
 }
 
 export class TileMetadataImagery {
-    imagery = new Map<string, TileMetadataImageryRecord>();
+    imagery = new Map<string, TileMetadataImageryRecordV1>();
 
     metadata: TileMetadataTableBase;
     constructor(metadata: TileMetadataTableBase) {
         this.metadata = metadata;
     }
 
-    public async get(imgId: string): Promise<TileMetadataImageryRecord> {
+    public async get(imgId: string): Promise<TileMetadataImageryRecordV1> {
         const existing = this.imagery.get(imgId);
         if (existing) return existing;
 
-        const item = await this.metadata.get<TileMetadataImageryRecord>(imgId);
+        const item = await this.metadata.get<TileMetadataImageryRecordV1>(imgId);
         if (item == null) throw new Error('Unable to find imagery: ' + imgId);
+        if (item.v == null) throw new Error('version 0 imagery no longer suppored');
```

Comment:
We need to support v0 for one release then we can patch it out.

Reply:
@blacha We don't if we migrate db before using

Reply:
cant migrate since the old version doesnt support the new version. so downtime!

and if we did migrate we dont need the if.

Reply:
Can now migrate before deployment https://github.com/linz/basemaps-team/pull/198